membership-common/src/main/scala/com/gu/memsub/ProductFamily.scala

```diff
@@ -206,6 +212,11 @@ object Benefit {
     override val isPhysical: Boolean = true
   }
 
+  object TierThree extends PaidMemberTier {
```

Comment:
it's not really a member tier (although this is all gone in  this PR)
https://github.com/guardian/members-data-api/pull/1079/files#diff-8434c9ffdcbb8fc26673d1f84012690fee7a77abf6ab033c73335a7b03a32f1aL141